src/libsync/propagateupload.h

```diff
@@ -320,6 +320,10 @@ class PropagateUploadFileV1 : public PropagateUploadFileCommon
 public:
     PropagateUploadFileV1(OwncloudPropagator *propagator, const SyncFileItemPtr &item)
         : PropagateUploadFileCommon(propagator, item)
+        , _startChunk(0)
+        , _currentChunk(0)
```

Comment:
Note: this could be initialized in the header file.

Reply:
Should I step in and fix it, or @ckamm ?

Reply:
Sorry, I missed the notifications for this. Looking now.

Reply:
@mrow4a (in general, feel free to just fix issues if I'm unresponsive!)